applications/ViewFi.md

```diff
@@ -0,0 +1,153 @@
+The platform will utilize Polkadot's parachain capabilities to ensure efficient token dispensing and NFT reward mechanisms. 
+
+### 🔍 Project Details
+
+**Tech Stack**
+
+Java, Spring, Postgres, React, Solidity, Forge, PReact and TypeScript. We use Infura for indexing and integrated into Wallet Connect SDK plus MetaKeep for wallet generation.
+
+**Core Documentation**
+
+- Video player widget with integrated reward mechanisms
+- Smart contracts for token dispensing and NFT rewards
+- Wallet infrastructure for seamless user onboarding
+- Analytics dashboard for campaign performance tracking
+- API for brand integration and campaign management
+
+Comprehensive documentation available at: https://docs.viewfi.io/
+
+**Product Deck:** https://drive.google.com/drive/u/0/folders/1i1DazxM9ASsGpHKg39sfJdSsDU3C7dXL
+
+**Project Limitation**
+- Will not provide traditional advertising network services
+- Will not store sensitive user data beyond necessary campaign metrics
+- Will not integrate onchain solutions beyond NFT creation (and distriution) and token airdrop.
+
+### 🧩 Ecosystem Fit
+
+ViewDrop fits into Polkadot's ecosystem as a DeFi application that bridges traditional marketing with web3 rewards, driving mainstream user adoption through familiar video content experiences enhanced with blockchain-based incentives.
+
+**Target Audience**
+
+Primary: Brands and marketers seeking innovative, ROI-driven advertising solutions
+Secondary: Web3 protocols wanting to grow their user base through rewarded video campaigns
+Tertiary: End users who want to be rewarded for their attention and engagement
+
+ViewFi addresses the critical need for privacy-compliant, engaging marketing solutions in the post-cookie era while simultaneously onboarding mainstream users to web3 through value exchange mechanisms.
+
+**Competitors**
+
+No direct competitors currently offer the same solution on any blockchain. Our unique approach combines video marketing, web3 rewards, and enterprise-grade partnerships, positioning us as the first-mover in this emerging category.
+
+## 👥 Team
+
+- **Team Name:** ViewFi
+- **Contact Name:** Marcus Junior
+- **Contact Email:** marcus@viewfi.io
+- **Website:** https://viewfi.io/
+
+### Team members
+
+Chris Gale (Founder & CEO)
+Marcus Lima (Head of Partnerships)
+Jon Hook (Advisor)
+Cristian Romero (Lead Developer)
+Dhiraj Kadam (Full Stack Developer)
+Moksedul Hoque (Full Stack Developer)
+Taimur Abid (UX/UI Designer)
+
+#### LinkedIn Profiles (if available)
+
+https://www.linkedin.com/in/chgale/
+https://www.linkedin.com/in/jonhook/
+https://www.linkedin.com/in/cristian-mgb/
+https://www.linkedin.com/in/codedem/
+https://www.linkedin.com/in/moksedul-hoque-uzzol-278aa0164/
+https://www.linkedin.com/in/taimur-abid-6b279a259/
+
+### Team Code Repos
+
+- https://github.com/{your_organisation}/{project_1}
+- https://github.com/{your_organisation}/{project_2}
```

Comment:
I would like to see the repositories of the team or the project or the team members